elliottlib/cli/create_textonly_cli.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import absolute_import, print_function, unicode_literals
+import datetime
+
+from errata_tool import Erratum, ErrataException
+from kerberos import GSSError
+import click
+
+from elliottlib.cli.common import cli
+from elliottlib import logutil, Runtime
+from elliottlib.exceptions import ElliottFatalError
+from elliottlib.util import exit_unauthorized, green_prefix, validate_release_date, \
+    YMD, validate_email_address
+import elliottlib
+
+LOGGER = logutil.getLogger(__name__)
+
+pass_runtime = click.make_pass_decorator(Runtime)
+
+#
+# Create Text only advisory
+# advisory:create-textonly
+
+
+@cli.command('create-textonly',
+             short_help='Create a textonly advisory along with notification bug attached')
```

Comment:
I am OK with a new command, but would prefer to integrate this into `create` command with a new impetus so that we can leverage the advisory boilerplate text in group.yml.

Reply:
I used to try to use a template in erratatool.yml but I realized in this use case for text-only we need to update almost every comment for each release since it's has different reasons, it's hardly create with boilerplate without change. Also the same reason to the related bug, so I thought the default value for jenkins job's parameters can treat as a boilerplate.

Reply:
For each text-only advisory we need to update the context about effected version, target version, related blocker bug and related blog/source link.